### ISSUE TITLE
Beholder-GAN: Generation and Beautification of Facial Images with Conditioning on Their Beauty Level

### ISSUE THREAD

**Issue by chullhwan-song** [chullhwan-song/Reading-Paper#81]
https://arxiv.org/abs/1902.02593

**Comment by chullhwan-song** [comment]
## Abstract
* Beauty is in the eye of the beholder - 아름다움은 보는 사람의 눈에 있다. 보는 사람마다 주간적?
* Beauty 를 Score로..
   * In the digital era, data-driven methods have been shown to be able to predict human-assigned beauty scores for facial images.
* beauty score가 주어진 얼굴을 generative
   * beauty score에 따른(condition) 얼굴이 생성된다는 의미.
* unsupervised beautification model
   * in the sense that it relies on no ground truth target images
## METHOD
### A GAN conditioned on a beauty score
* beauty score는 주관적인 속성이지만, 많은 사람들이 점수를 매기려는...
   * dataset 연구
      * "A benchmark for geometric facial beauty study"
      * "The beauty of capturing faces: Rating the quality of digital portraits"
      * " Assessing facial beauty through proportion analysis by image processing and supervised learning"
   * 이런 데이터를 이용하여 beauty score를 예측하려는 연구
      * "A new humanlike facial attractiveness predictor with cascaded fine-tuning deep learning model"
      * "Transferring rich deep features for facial beauty prediction"
      * "Beauty and countersignaling in online matching markets: Evidence from a randomized field experiment"
   * 나름 좋은 acc를 보였다.
* beauty score가 condition된 face image를 Generator(G)한다면, 
![image](https://user-images.githubusercontent.com/40360823/52621632-2b4a6980-2eeb-11e9-9a1e-d8f4942fbec9.png)
   * ![image](https://user-images.githubusercontent.com/40360823/52621648-3a311c00-2eeb-11e9-8003-5822d5f441ad.png)
   * ![image](https://user-images.githubusercontent.com/40360823/52621712-55039080-2eeb-11e9-92f2-3e05e2f8f095.png)는 latent space에서의 random Gaussian vector 
   * x 는 generator된 이미지.
* x가 정확한 beauty score에 의해 발생된 얼굴이미지인지 확증하기 위해서, discriminator D가 beauty level를 예측한다.
    * 여기서.. 중요한것은 D가 real vs fake를 구분하는것이 아닌듯.
![image](https://user-images.githubusercontent.com/40360823/52622022-2639ea00-2eec-11e9-84dc-80d2e549a359.png)
       * G loss :  ![image](https://user-images.githubusercontent.com/40360823/52622080-48336c80-2eec-11e9-8e7a-f8397190dcd2.png)
       * beauty score GT가 있어야 하지 않나?? - 위에서 unsupervised..??
       * 아무튼, 다수의 beholders에 의해 매긴 face beauty score가 있음 더 좋다는..
           *  we input not a single score but a vector of all the ratings available for the face.
* 이렇게 학습후에,  fixed z를 가진 Generator를 사용하고, 다른 레벨의 beauty score를 가진게 아니라(같은 score를 가졌다는 의미인듯~), 같은 사람에 속하는 Generator된 얼굴로부터 beauty score input β를 다양하게 하여 Face Generator할수 있다는 의미(다른 레벨을 갖는 face를 Generator).
  * After training, we can use the trained generator with some fixed z and vary the beauty score input β to generate faces belonging to the same person but having different beauty levels.


### Beautification
